src/coord/src/coord.rs

```diff
@@ -290,6 +292,13 @@ impl Coordinator {
                     // Inform the timestamper about this source.
                     self.update_timestamper(*id, true).await;
                     self.maybe_begin_caching(*id, &source.connector).await;
+                    // FIXME: I don't think making this start with num_workers copies of min is
```

Comment:
this comment no longer true -- all workers will explicitly drop or downgrade their upper frontiers